plugins/source-verifier/api.ts

```diff
@@ -1,16 +1,58 @@
-export interface Network {
-  name: 'ropsten' | 'mainnet' | 'rinkeby' | 'goerli'
-  id: number  
-}
-
 export interface VerifiedArtifacts {
   metadata: any
   contracts: any
 }
 
+export interface VerifyResult {
+  match: 'partial' | 'perfect' | 'no match',
+  success: boolean,
+  message: string
+}
+
+export interface VerifyData {
+  address: string,
+  chain: string | number,
+  files: Record<string, any> 
```

Comment:
one of the files is the metadata.json right?

Reply:
Yes